CMakeLists.txt

```diff
@@ -219,6 +219,23 @@ if (INSTALL_PKGCONFIG_MODULE)
 			set (EXTERNAL_XIPH_REQUIRE "${EXTERNAL_XIPH_REQUIRE} speex")
 		endif ()
 	endif ()
+	if (ENABLE_MPEG)
+		set (EXTERNAL_MPEG_REQUIRE "libmpg123")
+		get_filename_component(LAME_WE ${LAME_LIBRARY} NAME_WE)
+		if (LAME_HIP_LIBRARY)
+			get_filename_component(LAME_HIP_WE ${LAME_HIP_LIBRARY} NAME_WE)
+		endif ()
+		if (CMAKE_IMPORT_LIBRARY_PREFIX)
+			string (REGEX REPLACE "^${CMAKE_IMPORT_LIBRARY_PREFIX}" "" LAME_WE_NO_PREFIX ${LAME_WE})
+			if (LAME_HIP_LIBRARY)
+				string (REGEX REPLACE "^${CMAKE_IMPORT_LIBRARY_PREFIX}" "" LAME_HIP_WE_NO_PREFIX ${LAME_HIP_WE})
+			endif ()
+		endif ()
+		set (MPEG_LIBS "-l${LAME_WE_NO_PREFIX}")
+		if (LAME_HIP_LIBRARY)
+			set (MPEG_LIBS "${MPEG_LIBS} -l${LAME_HIP_WE}")
+		endif ()
```

Comment:
I'm unhappy with `MPEG_LIBS`: If you drop `libmpg123` as a `Requires.private` into the `.pc`, that obviates the need for specifying `MPEG_LIBS` in `Libs.private`. I strongly suggest you get rid of the `Libs.private` line.

Reply:
No, it is not libmpg123, it is LAME. It has no `.pc` file.

Reply:
oh sorry, missed that, :+1: